test/intl402/Intl/getCanonicalLocales/main.js

```diff
@@ -0,0 +1,48 @@
+// Copyright 2016 Mozilla Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+esid: sec-intl.getcanonicallocales
+description: Tests for existance and behavior of Intl.getCanonicalLocales
+info: |
+  8.2.1 Intl.getCanonicalLocales (locales)
+  1. Let ll be ? CanonicalizeLocaleList(locales).
+  2. Return CreateArrayFromList(ll).
+includes: [compareArray.js]
+---*/
+
+if (typeof Intl.getCanonicalLocales !== "function") {
+    $ERROR("Intl.getCanonicalLocales doesn't exist.");
+}
+
+if (Intl.getCanonicalLocales.length !== 1) {
+    $ERROR("Intl.getCanonicalLocales.length should be 1.");
+}
+
+let gCL = Intl.getCanonicalLocales;
+
+assertEq(Intl.getCanonicalLocales.length, 1);
```

Comment:
`assertEq` does not exist. You can use `assert.sameValue( value, expected [ , message ] );`